exchange_calendars/exchange_calendar_xtai.py

```diff
@@ -138,22 +160,24 @@ def bridge_fri(dt: datetime.datetime) -> datetime.datetime | None:
     to bridge the weekend and the official holiday.
     """
     dt += ONE_DAY
-    return dt if (dt.weekday() == FRIDAY and dt.year > 2013) else None
+    return dt if (dt.weekday() == FRIDAY and checker(dt)) else None
 
 
 NewYearsDay = new_years_day(observance=weekend_makeup)
 NewYearsDayExtraMon = new_years_day(observance=bridge_mon)
 NewYearsDayExtraFri = new_years_day(observance=bridge_fri)
 
+partial_bridge_mon = partial(bridge_mon, checker=check_between_2013_2024)
+partial_bridge_fri = partial(bridge_fri, checker=check_between_2013_2024)
 
 PeaceMemorialDay = Holiday(
     "Peace Memorial Day", month=2, day=28, observance=weekend_makeup
 )
 PeaceMemorialDayExtraMon = Holiday(
-    "Peace Memorial Day extra Monday", month=2, day=28, observance=bridge_mon
+    "Peace Memorial Day extra Monday", month=2, day=28, observance=partial_bridge_mon
```

Comment:
Could you provide a reference (within the PR comments is fine) confirming that this extra Monday is not applied from 2024.